### PR TITLE
Skeleton

### PR DESCRIPTION
**Pull request by Wiktor6**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    },
+    "require-dev": {
+        "friendsofphp/php-cs-fixer": "^3.4",
+        "matthiasnoback/symfony-config-test": "^4.3",
+        "phpro/grumphp": "^1.5.0",
+        "phpstan/phpstan": "^1.4",
+        "phpstan/phpstan-webmozart-assert": "^1.2",
+        "phpunit/phpunit": "^9.6",
+        "roave/security-advisories": "dev-master",
+        "symfony/phpunit-bridge": "6.2.*"
```
Comment (lukasz-falda) [inline composer.json]: update required for dev dependences in skeleton

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        "symfony/serializer": "^6.3",
+        "webmozart/assert": "^1.3"
+    },
+    "require-dev": {
+        "friendsofphp/php-cs-fixer": "^3.4",
+        "matthiasnoback/symfony-config-test": "^4.3",
+        "phpro/grumphp": "^1.5.0",
+        "phpstan/phpstan": "^1.4",
+        "phpstan/phpstan-webmozart-assert": "^1.2",
+        "phpunit/phpunit": "^9.6",
```
Comment (lukasz-falda) [inline composer.json]: phpunit with latest version?

**Review by lukasz-falda — APPROVED** [review]

File: src/DependencyInjection/Configuration.php [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+namespace Answear\FanCourier\DependencyInjection;
+
+use Symfony\Component\Config\Definition\Builder\TreeBuilder;
+use Symfony\Component\Config\Definition\ConfigurationInterface;
+
+class Configuration implements ConfigurationInterface
+{
+    public function getConfigTreeBuilder(): TreeBuilder
+    {
+        return new TreeBuilder('answear_boxnow');
```
Comment (ropczan) [inline src/DependencyInjection/Configuration.php:14]:
```suggestion
        return new TreeBuilder('answear_fancourier');
```

File: composer.json [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    "type": "symfony-bundle",
+    "license": "MIT",
+    "require": {
+        "php": "8.*",
+        "ext-json": "*",
+        "guzzlehttp/guzzle": "^7.7",
+        "psr/log": "^1.1",
+        "symfony/http-kernel": "^6.3",
+        "symfony/property-info": "^6.3",
+        "symfony/serializer": "^6.3",
```
Comment (lukasz-falda) [inline composer.json]: 6.3 for symfony? We are using 6.1

**Review by lukasz-falda — APPROVED** [review]

**Pull request merged**